### PR TITLE
Also retrieve class side methods in class search

### LINKED ISSUE #9: only instance methods are found in class search, class methods are not retrieved
As a developer I want to explore class methods just like instance methods in order to use algernon for all my code browsing.

# Acceptance Criteria
- [x] Class methods are shown in layer 2 of algernon results.
- [x] Add `(class)` prefix to class methods

### PR DESCRIPTION
**Pull request by Olliwehr**
* When searching inside a class (by navigating into it), now class methods are shown too.
* For clarification those results get a '(class)' prefix.
* closes #9

### CODE REVIEW

File: packages/Algernon.package/ALGDynamicWorldMenuType.class/methodProperties.json [hunk 1]
```diff
@@ -1,8 +1,6 @@
 {
 	"class" : {
-		"excludeDynamicWindows:" : "b 5/22/2021 20:46",
 		"extractMenuEntriesFrom:" : "NL 6/29/2021 15:00",
-		"filterOutDuplicates:" : "NM 6/29/2021 17:28",
-		"menuEntries:" : "NL 6/4/2021 16:21" },
+		"filterOutDuplicates:" : "NM 6/29/2021 17:28" },
```
Comment (Olliwehr) [inline packages/Algernon.package/ALGDynamicWorldMenuType.class/methodProperties.json:4]: Really don't know how these changes got in here. As they defintively belong to prior merged stuff, maybe this was not merged correctly? 🤔
But it didn't breakt sth here, soo... 😅

**Comment by Olliwehr** [comment]
I felt like i could "quickly" implement sth after the presentation 😅  
But sorry though for not following the pair programming.

File: packages/Algernon.package/ALGMethodType.class/class/algertemListWith..st [hunk 2]
```diff
@@ -1,4 +1,10 @@
...
 algertemListWith: aClass
-	^ aClass selectors asOrderedCollection
-		collect: [:each | ALGItem withAlgertype: (self withType:each withClass:aClass) ]
\ No newline at end of file
+	| items |
+	items := aClass selectors asOrderedCollection
+		collect: [:each | ALGItem withAlgertype: (self withType: each withClass: aClass) ].
+		
+	items addAll: (aClass class selectors asOrderedCollection
+		collect: [:each | ALGItem withAlgertype: (self withType: self prefixForClassMethods, each withClass: aClass class) ]).
```
Comment (niklasmohrin) [inline packages/Algernon.package/ALGMethodType.class/class/algertemListWith..st:8]: > `aClass class`

Wow, that is odd :laughing: (but correct)

File: packages/Algernon.package/ALGMethodType.class/instance/createListOfClassAwareAlgeractions..st [hunk 3]
```diff
@@ -0,0 +1,5 @@
+actions
+createListOfClassAwareAlgeractions: aMethod
+	methodClass
+		ifNil: [^ {}]
+		ifNotNil: [^ {self createActionBrowseExistingFor: aMethod. self createActionBrowseNewFor: aMethod}]
```
Comment (niklasmohrin) [inline packages/Algernon.package/ALGMethodType.class/instance/createListOfClassAwareAlgeractions..st]: nitpick: the return can be moved outside of the branches (also missing `self`; formatting up for debate):

```suggestion
^ self methodClass
    ifNil: [ {} ]
    ifNotNil: [ {
        self createActionBrowseExistingFor: aMethod .
        self createActionBrowseNewFor: aMethod        } ]
```

Comment (Olliwehr) [reply packages/Algernon.package/ALGMethodType.class/instance/createListOfClassAwareAlgeractions..st]: @niklasmohrin i would directly commit your suggestion from here, but it seems like the method body then lacks one indent or is that caused by the view in github?

File: packages/Algernon.package/ALGMethodType.class/class/algertemListWith..st (hunk 2 above)
Comment (Olliwehr) [reply packages/Algernon.package/ALGMethodType.class/class/algertemListWith..st:8]: Yeah, quite strange 😄 But as far as my understanding goes `aClass class` then refers to the meta-class of `aClass` (which internally handles the method dictionary for the class side of `aClass`).

File: packages/Algernon.package/ALGMethodType.class/instance/createListOfClassAwareAlgeractions..st (hunk 3 above)
Comment (niklasmohrin) [reply packages/Algernon.package/ALGMethodType.class/instance/createListOfClassAwareAlgeractions..st]: oh no, I didn't write that properly :D

**Review by TheGrayStone — APPROVED** [review]
Looks all good to me now!

**Pull request merged**